portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Sherry's Portfolio</title>
```

Comment:
Optional: This could be `Sherry's Portfolio | Gallery` or something similar

Reply:
done!